src/components/input.js

```diff
@@ -252,10 +260,11 @@ const withAutocomplete = WrappedComponent => ({
     inputId: id,
     labelId
   }, [
-    ({ getInputProps, getMenuProps, getItemProps, isOpen, openMenu, toggleMenu, highlightedIndex }) => {
+    ({ getInputProps, getMenuProps, getItemProps, isOpen, openMenu, toggleMenu, clearSelection, highlightedIndex }) => {
+      clearMethod = clearSelection
```

Comment:
Is it possible to attach a ref to the downshift and call `clearSelection` through that?

Reply:
I honestly don't know how to do that, since we're using this method in the event listener on the input, we need this method before the downshift block that exposes this method is available